datanode/gateway/graphql/schema.graphql

```diff
@@ -5096,15 +5096,33 @@ type UpdateReferralProgram {
   stakingTiers: [StakingTier!]!
 }
 
+type RewardFactors {
+  "The proportion of the referee's taker infrastructure fees to be rewarded to the referrer"
+  infrastructureFactor: String!
+  "The proportion of the referee's taker maker fees to be rewarded to the referrer"
+  makerFactor: String!
+  "The proportion of the referee's taker liquidity fees to be rewarded to the referrer"
+  liquidityFactor: String!
+}
+
+type DiscountFactors {
+  "The proportion of the referee's taker infrastructure fees to be discounted"
+  infrastructureFactor: String!
+  "The proportion of the referee's taker maker fees to be discounted"
+  makerFactor: String!
+  "The proportion of the referee's taker liquidity fees to be discounted"
+  liquidityFactor: String!
+}
+
 type BenefitTier {
   "The minimum number of epochs the party needs to be in the referral set to be eligible for the benefit"
   minimumEpochs: Int!
   "The minimum running notional for the given benefit tier"
   minimumRunningNotionalTakerVolume: String!
   "The proportion of the referee's taker fees to be discounted"
-  referralDiscountFactor: String!
+  referrals: DiscountFactors!
   "The proportion of the referee's taker fees to be rewarded to the referrer"
-  referralRewardFactor: String!
+  referralRewardFactors: RewardFactors!
```

Comment:
@ze97286 Is it possible to deprecate the old fields marking with `@deprecated` rather than directly changing them?